salt/search/init.sls

```diff
@@ -76,6 +76,7 @@ search-console-ready:
         - name: ./bin/console --env={{ pillar.elife.env }}
         - cwd: /srv/search
         - user: {{ pillar.elife.deploy_user.username }}
+        - timeout: 5 # seconds
         - require:
             - search-composer-install
             - aws-credentials
```

Comment:
I think requiring `gearman-server` is needed here, although it may naturally work due to the `.top` file ordering

Reply:
lets make it explicit